### PR TITLE
docs: populate docs with `ibis-examples` repo content

### PR DESCRIPTION
**Pull request by cpcloud**
Moves content except for `Substrait.ipynb` to a new `Examples` tab in our documentation. `Substrait.ipynb` has multiple issues which need to be addressed if we want to include it. I opted for not including it at the moment.

### CODE REVIEW

File: docs/_quarto.yml [hunk 1]
```diff
@@ -125,6 +125,16 @@ website:
...
+
+    - id: examples
+      title: "Examples"
+      style: "docked"
+      collapse-level: 2
+      contents:
+        - auto: "examples/parquet-and-duckdb.qmd"
+        - auto: "examples/clickhouse-hackernews.qmd"
+        - auto: "examples/imdb.qmd"
+
```
Comment (lostmygithubaccount) [inline docs/_quarto.yml:137]: this isn't showing up at the top like I assume we want, need to add in the navbar left/right thing above

**Comment by gforsyth** [comment]
> * DuckDB's substrait extension doesn't seem to understand the annotated substrait functions like `avg:fp64` or `count:any`. Not sure if we're generating incorrect protobufs, or the extension needs to be updated to handle them.

it's an issue with the extension, although I thought they had fixed it.
https://github.com/duckdb/substrait/issues/65

Might be they haven't released the updated extension?

**Comment by cpcloud** [comment]
> should these just fit elsewhere in the website? tutorials/how-to guides/blog posts?

I think anywhere but blogs is fine. I think these are probably good to go in how-tos maybe in a data analysis subsection (?): not long enough for a tutorial and usually specific to a problem or dataset.

**Comment by cpcloud** [comment]
Moved:

- duckdb-parquet to `docs/how-to/input-output`
- clickhouse-hackernews to `docs/data-platforms/clickhouse-hackernews.qmd` (I'm going to change this to be just `clickhouse`)
- imdb to `docs/how-to/analytics/imdb`

File: docs/_quarto.yml (hunk 1 above)
Comment (cpcloud) [reply docs/_quarto.yml]: Rolled the new docs files into the existing structure.

**Comment by cpcloud** [comment]
@ncclementi @gforsyth @jcrist Any thoughts here?

We should add the substrait example back in once we get it working, but ideally we can merge this in in the mean time.

**Review by gforsyth — APPROVED** [review]
Yeah, I'm good with more things in `how-to`

File: docs/examples/clickhouse-hackernews.qmd [hunk 2]
```diff
@@ -0,0 +1,142 @@
+---
+title: Using Ibis with ClickHouse
+---
+
+[Ibis](https://ibis-project.com) supports reading and querying data using
+[ClickHouse](https://clickhouse.com/) as a backend.
+
+In this example we'll demonstrate using Ibis to connect to a ClickHouse server,
+and executing a few queries.
```
[the remark this answers is not in the document]
Comment (gforsyth) [reply docs/examples/clickhouse-hackernews.qmd]: oops, forgot to submit that review.  no kind of blocker

[the remark this answers is not in the document]
Comment (cpcloud) [reply docs/examples/clickhouse-hackernews.qmd]: Done!

**Review by lostmygithubaccount — APPROVED** [review]

**Pull request merged**